CMakeLists.txt

```diff
@@ -0,0 +1,86 @@
+cmake_minimum_required(VERSION 3.0)
+
+# set cmake module path, to search in cmake/modules first
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/modules")
+#-----------------------------------------------------------------------------
+#                           GENERAL CONFIGURATION
+#-----------------------------------------------------------------------------
+project(LibVMI VERSION 0.13.0 LANGUAGES C)
```

Comment:
Another hesitation I have is regarding the project version.
The library produces by CMake will be `libvmi.so.0.13.0`
But we want `libvmi.so.0.0.13`
Should we change the project version to match the library name we want to get ?